docs/gds-supported-platform/getting-started-gsp.md

```diff
@@ -0,0 +1,167 @@
+
+## Create a Helm chart
+
+The GDS Supported Platform uses a packaging format called [Helm charts](https://helm.sh/docs/developing_charts/). A chart is a collection of files that describe a related set of Kubernetes resources.
+
+You create Helm charts as files in a directory. These files are then packaged into versioned archives that users can deploy.
+
+1. Create a root directory in your GitHub repository. This directory will contain the chart.
+
+1. Create a `Chart.yaml` file in the root directory with the following code:
+
+    ```
+    apiVersion: v1
+    appVersion: "1.0"
+    description: CHART_DESCRIPTION
+    name: CHART_NAME
+    version: 0.1.0
+    ```
+
+    This file defines metadata about the chart.
+
+1. Create a `templates` directory in the root directory. This directory contains all Kubernetes object definitions.
+
+1. Create a `values.yaml` file in the root directory. This file sets the default values for your desired chart variables.
+
+## Create a Kubernetes Deployment object
+
+You run an app by creating a [Kubernetes Deployment object](https://kubernetes.io/docs/concepts/#kubernetes-objects). This object defines your app and its routes, databases and all other relevant information. You describe a Deployment in a YAML file.
+
+1. Create a `deployment.yaml` file in the `templates` directory. The following example uses an [nginx](https://hub.docker.com/_/nginx/) container image called `myapp`. Replace this nginx container image with your app image:
+
+    ```
+    apiVersion: apps/v1beta2
+    kind: Deployment
+    metadata:
+      name: {{ .Release.Name }}-myapp
+      labels:
+        app.kubernetes.io/name: myapp
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      replicas: {{ .Values.replicas }}
+      selector:
+        matchLabels:
+          app.kubernetes.io/name: myapp
+          app.kubernetes.io/instance: {{ .Release.Name }}
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/name: myapp
+            app.kubernetes.io/instance: {{ .Release.Name }}
+        spec:
+          containers:
+            - name: myappcontainer
+              image: "nginx:latest" #Replace this with your app image
+              ports:
+                - name: http
+                  containerPort: 80
+                  protocol: TCP
+    ```
+
+    Helm automatically populates the `{{ .Release.Name }}` and `{{ .Values.replicas }}` variables when you render the chart.
+
+1. Run the following command in the root directory to render the chart:
+
+    ```
+    helm template --name example .
+    ```
+
+1. Check `stdout` to see if the chart rendered correctly.
+
+## Create a service
+
+By default, your apps are not accessible to the public. To expose them to the public, you must set up a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) and an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) into the Kubernetes cluster.
```

Comment:
A service is also required for internal load balancing (or, to put it another way, to allow other pods to connect to it; it's not just a "public" / "external" thing). Do we need to make this clearer?

Reply:
I think we might end up duplicating the kubernetes docs if we expand too much ... but I agree we should mention the analogy to a mini internal loadbalancer as that could help solidify it's purpose